viziphant/unitary_event_analysis.py

```diff
@@ -180,203 +150,227 @@ def plot_ue(data, Js_dict, sig_level, binsize, winsize, winstep,
+        import numpy as np
+        import quantities as pq
 
-    N = len(data[0])
+        import viziphant
+        from elephant.spike_train_generation import homogeneous_poisson_process
+        from elephant.unitary_event_analysis import jointJ_window_analysis
 
-    t_winpos = ue._winpos(t_start, t_stop, winsize, winstep)
-    Js_sig = ue.jointJ(sig_level)
-    num_tr = len(data)
+        np.random.seed(10)
+
+        spiketrains1 = [homogeneous_poisson_process(rate=20 * pq.Hz,
+                        t_stop=2 * pq.s) for _ in range(5)]
+        spiketrains2 = [homogeneous_poisson_process(rate=50 * pq.Hz,
+                        t_stop=2 * pq.s) for _ in range(5)]
+
+        spiketrains = np.stack((spiketrains1, spiketrains2), axis=1)
+        ue_dict = jointJ_window_analysis(spiketrains,
+                                         bin_size=5 * pq.ms,
+                                         win_size=100 * pq.ms,
+                                         win_step=10 * pq.ms)
+        viziphant.unitary_event_analysis.plot_ue(spiketrains, Js_dict=ue_dict,
+                                                 significance_level=0.34,
+                                                 unit_real_ids=['1', '2'])
+        plt.show()
+
+    Refer to `UEA Tutorial <https://elephant.readthedocs.io/en/latest/
+    tutorials/unitary_event_analysis.html>`_ for real-case scenario.
+    """
+    n_trials = len(spiketrains)
+    n_neurons = len(spiketrains[0])
+
+    input_parameters = Js_dict['input_parameters']
+    t_start = input_parameters['t_start']
+    t_stop = input_parameters['t_stop']
+    bin_size = input_parameters['bin_size']
+    win_size = input_parameters['win_size']
+    win_step = input_parameters['win_step']
+    pattern_hash = input_parameters['pattern_hash']
+    if len(pattern_hash) > 1:
+        raise ValueError(f"To not clutter the plots, only one pattern hash is "
+                         f"required; got {pattern_hash}. You can call this "
+                         f"function multiple times for each hash at a time.")
+    for key in ['Js', 'n_emp', 'n_exp', 'rate_avg']:
+        Js_dict[key] = Js_dict[key].squeeze()
+    neurons_participated = ue.inverse_hash_from_pattern(pattern_hash,
+                                                        N=n_neurons).squeeze()
+
+    t_winpos = ue._winpos(t_start=t_start, t_stop=t_stop, win_size=win_size,
+                          win_step=win_step)
+    Js_sig = ue.jointJ(significance_level)
 
     # figure format
+    plot_params_user = plot_params
     plot_params = plot_params_default.copy()
     plot_params.update(plot_params_user)
-    if len(plot_params['unit_real_ids']) != N:
+    if plot_params['unit_real_ids'] is None:
+        plot_params['unit_real_ids'] = ['not specified'] * n_neurons
+    if len(plot_params['unit_real_ids']) != n_neurons:
         raise ValueError('length of unit_ids should be' +
                          'equal to number of neurons!')
```

Comment:
```suggestion
        raise ValueError('length of unit_ids should be ' +
                         'equal to number of neurons!')
```
Inserted a space, since it prints "beequal".